cms/static/cms/js/require-config.js

```diff
@@ -247,7 +247,7 @@
                 exports: 'XBlock',
                 deps: ['xblock/core']
             },
-            'coffee/src/main': {
+            'js/main': {
```

Comment:
Would you mind moving this beneath the ``cms`` namespace directory, so this would the be:

    'cms/js/main': {

This reduces the risk of conflicts, and also seems clearer as it is the main file for CMS. I'm moving us to always putting code into a namespace directory.